src/cryptoadvance/specter/devices/hwi/jade.py

```diff
@@ -657,7 +662,10 @@ def can_sign_taproot(self) -> bool:
 
 
 def enumerate(
-    password: Optional[str] = None, expert: bool = False, chain: Chain = Chain.MAIN
+    password: Optional[str] = None,
+    expert: bool = False,
+    chain: Chain = Chain.MAIN,
+    unlock=True,
```

Comment:
Maybe 'unlock=False' is the better option here, and then do something similar/consistent with ledger and trezor if the jade is not unlocked ?  Your call ofc - I am aware that 'unlock=True' will mean existing callers behaviour will be unchanged, so ok maybe that's a more cautious approach.  I'll leave it up to you ofc ;-)

Reply:
I think it is easier to always auth by default and only skip it for certain scenarios, like the startup of Specter ...

Reply:
Change to `skip_unlocking`